index.src.html

```diff
@@ -70,6 +70,9 @@
       text: Navigator
 spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
+    text: same-origin; url: concept-origin-effective-domain
```

Comment:
Thanks for making me aware of the right fragment of the URL.
When I remove this line completely, I get:
LINK ERROR: No 'dfn' refs found for 'same-origin' that are marked for export.
&lt;a data-link-type="dfn" data-lt="same-origin"&gt;same-origin&lt;/a&gt;
Does it mean I should keep this line?